implementations/rust/ockam/ockam_api/src/cli_state/journeys/journeys.rs

```diff
@@ -195,92 +200,181 @@ impl CliState {
     }
 
     /// Return a list of journeys for which we want to add spans
```

Comment:
Why do we write to host and project journey at the same time?

Reply:
The two journeys end-up being different:

 - The project journey groups events possibly coming from different machines from different users.
 - The host journey is machine centric and can contain events that are emitted before a project even exists (when toying around with local nodes for example).